src/main/java/ben/tasks/TaskList.java

```diff
@@ -0,0 +1,201 @@
+package ben.tasks;
+
+import ben.commands.DeadlineCommand;
+import ben.exceptions.BenException;
+import ben.ui.Ui;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents a list of tasks in the Ben task management application.
+ */
+public class TaskList {
+
+    private List<Task> tasks;
```

Comment:
I like that you used plural form on names representing a collection of objects 👍